### PR TITLE
Add command to bump job's lifespan

### PR DESCRIPTION
**Pull request by romasku**
Closes: https://github.com/neuro-inc/platform-client-python/issues/2008
Server side PR: https://github.com/neuro-inc/platform-api/pull/1514

### CODE REVIEW

File: neuro-cli/src/neuro_cli/formatters/jobs.py [hunk 1]
```diff
@@ -269,6 +276,21 @@ def __call__(self, job_status: JobDescription) -> RenderableType:
...
+        self._datetime_formatter = datetime_formatter
+
+    def __call__(self, job_status: JobDescription) -> RenderableType:
+        res = (
+            f"Updated job {job_status.id} life span to "
+            f"be {format_life_span(job_status.life_span)}"
+        )
+        life_span_ends = get_lifespan_ends(job_status)
+        if life_span_ends is not None:
+            res += f"\nLife ends (approx): {self._datetime_formatter(life_span_ends)}"
```
Comment (asvetlov) [inline neuro-cli/src/neuro_cli/formatters/jobs.py:290]:
```suggestion
            res += f"\nLife span ends (approx): {self._datetime_formatter(life_span_ends)}"
```
To be in sync with the text above

File: neuro-cli/src/neuro_cli/job.py [hunk 2]
```diff
@@ -461,6 +462,35 @@ async def status(root: Root, job: str, full_uri: bool) -> None:
...
+@command()
+@argument("job", type=JOB)
+@argument(
+    "additional-life-span",
+    type=str,
+    metavar="TIMEDELTA",
+)
+async def bump_life_span(root: Root, job: str, additional_life_span: str) -> None:
+    """
+    Display status of a job.
```
Comment (asvetlov) [inline neuro-cli/src/neuro_cli/job.py:474]: The command description is not correct I believe, please fix.

File: neuro-sdk/src/neuro_sdk/jobs.py [hunk 3]
```diff
@@ -439,6 +439,16 @@ async def kill(self, id: str) -> None:
             # an error is raised for status >= 400
             return None  # 201 status code
 
+    async def bump_life_span(self, id: str, additional_life_span: float) -> None:
```
Comment (asvetlov) [inline neuro-sdk/src/neuro_sdk/jobs.py:442]: Document new method in api docs please.

**Review by asvetlov — CHANGES_REQUESTED** [review]
looks good, please fix minor issues.

**Comment by romasku** [comment]
Sorry for those issues, forgot to recheck PR by myself. Should be fixed now.

**Review by asvetlov — COMMENTED** [review]
LGTM, thanks!

**Pull request merged**